src/DynamoCoreWpf/UI/Prompts/PresetStatePrompt.xaml.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Windows;
+using Dynamo.Controls;
+using Dynamo.Utilities;
+using System;
+
+namespace Dynamo.Nodes
+{
+    /// <summary>
+    /// Interaction logic for PresetStatePrompt.xaml
+    /// </summary>
+    public partial class PresetStatePrompt : Window
```

Comment:
Sorry to change this - isn't this just `PresetPrompt`?  Are there multiple states of a single `Preset`?